packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -595,6 +598,48 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupSlasPrivateClientProxy(app, options) {
+        if (options.useSlasPrivateClient) {
+            const clientSecret = process?.env?.SLAS_PRIVATE_CLIENT_SECRET
+            if (!clientSecret) {
+                app.use('/ssr/auth', (_, res) => {
+                    return res.status(501).json({
```

Comment:
Nice thinking!

Would you consider crashing the whole dev server and logging in addition to this? If folks choose this configuration and don't set the environment variable, then things aren't going to work and it would nice to fail as loudly as we can.